src/com/lftechnology/java/training/alina/basic/packagejarexecute/NumberAdd.java

```diff
@@ -0,0 +1,68 @@
+
+package com.lftechnology.java.training.alina.basic.packagejarexecute;
+
+import java.util.Scanner;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * <p>NumberAdd Class is used to display the sum of two numbers</p>
+ * 
+ * <p>Steps : </p>
+ * <ol type="a">
+ * <li>Create a Java application AddNumber, which takes two number from User and prints sum.</li>
+ * <li>Create Executable Jar</li>
+ * <li>Run java application from command line which should take values from user.</li>
+ * <li>Print sum</li>
+ * <li>Create JavaDoc of Application</li>
+ * </ol>
+ * 
+ * @author Alina Shakya <alinashakya@lftechnology.com>
+ */
+public class NumberAdd {
+
+	private static final Logger LOGGER =
+		Logger.getLogger(NumberAdd.class.getName());
+
+	/**
+	 * Main method used to calculate and display the sum of two numbers
+	 * 
+	 * @param args
+	 * @author Alina Shakya <alinashakya@lftechnology.com>
+	 */
+	public static void main(String[] args) {
+
+		NumberValidation numValidation = new NumberValidation();
+		try (Scanner input = new Scanner(System.in)) {
+			float firstNumber =
+				numValidation.numberValidation(input, "Enter first number : ");
+			float secondNumber =
+				numValidation.numberValidation(input, "Enter second number : ");
+			float sumOfTwoNumbers =
+				calculateSumOfNumbers(firstNumber, secondNumber);
+			LOGGER.log(
+				Level.INFO, "Sum of two numbers is : {0}", sumOfTwoNumbers);
+		}
+		catch (Exception e) {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1166)